src/test/multi_stress_watch.cc

```diff
@@ -144,17 +144,24 @@ int main(int args, char **argv)
     std::cerr << "Error " << ret << " in cluster.init" << std::endl;
     return ret;
   }
+
```

Comment:
could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes, in this case, it would be "test/multi_stress_watch: ".